### PR TITLE
Chores/fix broken rspec tests #160794897

### PR DESCRIPTION
**Pull request by fddayan**

### CODE REVIEW

File: spec/vcr/applications_controller/show.yml [hunk 1]
```diff
@@ -85,12 +85,12 @@ http_interactions:
       Cache-Control:
       - no-cache,must-revalidate,max-age=0,no-store,private
       Set-Cookie:
-      - BrowserId=a1NKcUKLQ0KnQiGoVFDdJQ;Path=/;Domain=.salesforce.com;Expires=Mon,
-        19-Nov-2018 18:24:27 GMT;Max-Age=5184000
+      - BrowserId=4g9nYTa4SP2CGLitfJInOQ;Path=/;Domain=.salesforce.com;Expires=Sun,
+        25-Nov-2018 15:42:38 GMT;Max-Age=5184000
       Expires:
       - Thu, 01 Jan 1970 00:00:00 GMT
       Sforce-Limit-Info:
```
Comment (akegan) [inline spec/vcr/applications_controller/show.yml:92]: wow this is cool that we get this info back on a request. I wonder if that means we could track it externally somehow. cc @callaghanc

**Review by akegan — APPROVED** [review]
looks great!  😄

File: spec/vcr/applications_controller/show.yml (hunk 1 above)
Comment (fddayan) [reply spec/vcr/applications_controller/show.yml:92]: yes, I saw that too. pretty cool

**Comment by fddayan** [comment]
@callaghanc do you want to take a look at the VCR cassette before merging into master?

File: spec/vcr/applications_controller/show.yml [hunk 2]
```diff
@@ -99,7 +99,7 @@ http_interactions:
       - chunked
     body:
       encoding: UTF-8
-      string: '{"totalSize":1,"done":true,"records":[{"attributes":{"type":"Application__c","url":"/services/data/v43.0/sobjects/Application__c/a0o0U000000VVw8QAG"},"Id":"a0o0U000000VVw8QAG","Name":"APP-00262254","Applicant__c":"a0n0U0000030tBrQAI","Alternate_Contact__c":null,"Listing__c":"a0W0U000000MX4vUAG","Listing__r":{"attributes":{"type":"Listing__c","url":"/services/data/v43.0/sobjects/Listing__c/a0W0U000000MX4vUAG"},"Name":"LAP
+      string: '{"totalSize":1,"done":true,"records":[{"attributes":{"type":"Application__c","url":"/services/data/v43.0/sobjects/Application__c/a0o0U000000VVw8QAG"},"Id":"a0o0U000000VVw8QAG","Name":"APP-00262254","Applicant__c":"a0n0U0000030tBrQAI","Primary_Applicant__c":"0030U00000P8lxaQAB","Alternate_Contact__c":null,"Listing__c":"a0W0U000000MX4vUAG","Listing__r":{"attributes":{"type":"Listing__c","url":"/services/data/v43.0/sobjects/Listing__c/a0W0U000000MX4vUAG"},"Name":"LAP
```
Comment (akegan) [inline spec/vcr/applications_controller/show.yml:102]: Primary_applicant__c is the only piece that's new potential user info and it's definitely fake. "Lappy Laptest" https://sfhousing--full.cs97.my.salesforce.com/0030U00000P8lxaQAB

File: spec/vcr/applications_controller/show.yml (hunk 1 above)
Comment (callaghanc) [reply spec/vcr/applications_controller/show.yml:92]: @akegan for sure! anything we're getting in responses is potentially able to be used in logging or record-keeping. did you have some ideas about how you want to use this data? e.g. GA?

**Pull request merged**